@utahdts/utah-design-system/react/components/buttons/ConfirmationButton.jsx

```diff
@@ -59,9 +59,9 @@ function ConfirmationButton({
   size,
   type,
 }) {
-  const [isClicked, setClicked] = useState(false);
+  const [isClicked, setIsClicked] = useState(false);
   const buttonClick = () => {
-    setClicked(!isClicked);
+    setIsClicked(!isClicked);
```

Comment:
may be more correct to check previous state in case `isClicked` it's a stale scope variable?
```setIsClicked((previousIsClicked) => !previousIsClicked);```

Reply:
UDS-1289